tdrs-backend/tdpservice/scheduling/management/db_backup.py

```diff
@@ -268,28 +248,47 @@ def get_database_credentials(database_uri):
     database_name = database_uri
     return [username, password, host, port, database_name]
 
-
-def main(argv, sys_values, system_user):
-    """Handle commandline args."""
-    arg_file = "/tmp/backup.pg"
-    arg_database = sys_values['DATABASE_URI']
+def get_opts(argv, db_name):
+    """Parse command line options."""
+    arg_file = f"/tmp/{db_name}_backup.pg"
```

Comment:
this has replaced our previous backup in prod (backup.pg) with the db_name hash. is this what was intended?